app/src/main/AndroidManifest.xml

```diff
@@ -189,6 +189,28 @@
                 android:value="true" />
         </service>
 
+        <service
+            android:name="com.x8bit.bitwarden.ui.platform.feature.tiles.GeneratorTileService"
```

Comment:
The declared service name must match the service name from the MAUI app. If this service name doesn't match, it creates a frustrating user experience upon upgrading. 

Check `BitwardenAppComponentFactory` for examples mapping Autofill and CredentialProvider services to their correct classes.

Same applies to `MyVaultTileService`, below.